zkstack_cli/crates/config/src/consts.rs

```diff
@@ -1,3 +1,7 @@
+use ethers::abi::Address;
+use zksync_basic_types::U256;
+use zksync_utils::u256_to_account_address;
```

Comment:
it seems `zksync_utils` was imported only for `u256_to_account_address`, I would prefer to simply copy this one function and remove import